app/scripts/models/auth_brokers/fx-fennec-v1.js

```diff
@@ -8,12 +8,49 @@
  */
 
 define([
-  'models/auth_brokers/fx-desktop-v2'
-], function (FxDesktopV2AuthenticationBroker) {
+  'models/auth_brokers/fx-desktop-v2',
+  'underscore'
+], function (FxDesktopV2AuthenticationBroker, _) {
   'use strict';
 
+  var proto = FxDesktopV2AuthenticationBroker.prototype;
+
   var FxFennecV1AuthenticationBroker = FxDesktopV2AuthenticationBroker.extend({
-    type: 'fx-fennec-v1'
+    type: 'fx-fennec-v1',
+
+    commands: _.extend({}, proto.commands, {
+      SYNC_PREFERENCES: 'fxaccounts:sync_preferences'
+    }),
+
+    defaultCapabilities: _.extend({}, proto.defaultCapabilities, {
+      emailVerificationMarketingSnippet: false,
+      syncPreferencesNotification: true
+    }),
+
+    defaultBehaviors: _.extend({}, proto.defaultBehaviors, {
+      afterForceAuth: {
+        halt: false,
+        redirectTo: 'force_auth_complete'
+      },
+      afterResetPasswordConfirmationPoll: {
+        halt: false
+      },
+      afterSignIn: {
+        halt: false,
+        redirectTo: 'signin_complete'
+      },
+      afterSignUpConfirmationPoll: {
+        halt: false,
+        redirectTo: 'signup_complete'
+      },
+      beforeSignUpConfirmationPoll: {
+        halt: false
+      }
+    }),
+
+    notifyOfSyncPreferences: function () {
```

Comment:
Change this to `openSyncPreferences` and document it. Add a stub to the BaseAuthenticationBroker too.